src/main/java/javaintern/entity/Reservation.java

```diff
@@ -0,0 +1,144 @@
+package javaintern.entity;
+
+import javaintern.constant.ReservationState;
+
+import java.time.LocalDateTime;
+import java.util.Date;
+
+public class Reservation {
+	private Integer id;
+	private Integer organizationId;
+	private Integer agentId;
+	private Integer customerId;
+	private Integer productId;
+	private ReservationState state;
+	private LocalDateTime fromDate;
+	private LocalDateTime toDate;
+	private Double price;
+	private Double quote;
+	private String currency;
+	private Integer guests;
+	private String notes;
+	private Date version;
+
+	public Reservation(Integer id, Integer organizationId, Integer agentId, Integer customerId, Integer productId, ReservationState state) {
```

Comment:
why these parameters?

Reply:
I'm not sure what parameters i need here. I'll change it to public Reservation() {}